client/src/env.ts

```diff
@@ -55,6 +55,14 @@ export const UPDATES_BASE_URL =
   process.env.REACT_APP_UPDATES_BASE_URL ||
   "https://updates.developer.allizom.org";
 
+export const GLEAN_CHANNEL = process.env.REACT_APP_GLEAN_CHANNEL || "stage";
+export const GLEAN_DEBUG = Boolean(
+  JSON.parse(process.env.REACT_APP_GLEAN_DEBUG || "false")
+);
+export const GLEAN_DISABLED = Boolean(
+  JSON.parse(process.env.REACT_APP_GLEAN_DISABLED || "true")
+);
```

Comment:
Does this mean Glean is disabled unless `REACT_APP_GLEAN_DISABLED` is set to `false`? Why not have `GLEAN_ENABLED` that defaults to `false`? IMO flags should have a default value of `false`.